src/DataViews/LiveFunctionsDataViewTest.cpp

```diff
@@ -384,6 +386,34 @@ TEST_F(LiveFunctionsDataViewTest, ContextMenuActionsAreInvoked) {
     CheckExportToCsvIsInvoked(context_menu, app_, view_, expected_contents);
   }
 
+  // Export events to CSV
+  {
+    const uint32_t kThreadId = 111;
+    const std::string kThreadName = "Test Thread";
+    capture_data_->AddOrAssignThreadName(kThreadId, kThreadName);
+    EXPECT_CALL(app_, GetCaptureData).WillRepeatedly(testing::ReturnRef(*capture_data_));
+
+    constexpr uint64_t kStart = 12345;
+    constexpr uint64_t kEnd = 67890;
+    TimerInfo timer;
+    timer.set_start(kStart);
+    timer.set_end(kEnd);
+    timer.set_thread_id(kThreadId);
+    std::vector<const TimerInfo*> timers = {&timer};
+    EXPECT_CALL(app_, GetAllFunctionCalls).WillRepeatedly(testing::Return(timers));
+
+    std::string expected_contents =
+        absl::StrFormat(R"#("Name","Thread","Start","End","Duration (ns)")#"
```

Comment:
Why these `#`?

Reply:
We have character`()` in the raw string (i.e.,`"Duration (ns)"`). Hence we use `#` as an escape character here.

go/totw/64

Reply:
Ah gotcha! The parentheses, it did not realize this. Thank you!